parselmouth/adapters/dfp/tests/config_tests.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Acceptance checks to determine if API responses are compatible with the
+current interface implementation
+"""
+
+# Future-proof
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+# Standard Library Imports
+import os.path
+import unittest
+
+# Parselmouth imports
+from parselmouth import __file__ as PARSELTOUNGE_PACKAGE_PATH
+from parselmouth.config import ParselmouthConfig
+from parselmouth.adapters.dfp.config import DFPConfig
+from parselmouth.exceptions import ParselmouthException
+from parselmouth.constants import ParselmouthProviders
+
+
+class TestParseltoungDFPConfig(unittest.TestCase):
+
+    def setUp(self):
+        # Dummy credentials
+        self.credentials = {
+            'client_id': 'ID',
+            'client_secret': 'SECRET',
+            'refresh_token': 'REFRESH_TOKEN',
+            'application_name': 'APPLICATION_NAME',
+            'network_code': 'NETWORK_CODE',
+        }
+
+        # Dummy credentials file
+        self.config_path = os.path.join(
```

Comment:
Should we consider moving these tests into the `tests/` directory of the dfp adapter?